GVFS/GVFS.Common/RetryConfig.cs

```diff
@@ -134,8 +134,9 @@ private static bool TryGetFromGitConfig(GitProcess git, string configName, int d
             value = defaultValue;
             error = string.Empty;
 
+            System.Diagnostics.Debugger.Launch();

```

Comment:
Don't think we want to keep this in production code. :)

Reply:
I'm sure it'll be fine :)

Thanks for catching that...